schematron/v2.2.0/tests/data/HVAC_example1.xml

```diff
@@ -130,15 +131,94 @@
                         </auc:CoolingSources>
                         <auc:Deliveries>
                             <auc:Delivery ID="Delivery-A">
+                                <auc:DeliveryType>
+                                    <auc:CentralAirDistribution>
+                                        <auc:AirDeliveryType>Central fan</auc:AirDeliveryType>
+                                        <auc:TerminalUnit>CAV terminal box no reheat</auc:TerminalUnit>
+                                        <auc:ReheatSource>Heating plant</auc:ReheatSource>
+                                        <auc:ReheatControlMethod>Single Maximum</auc:ReheatControlMethod>
+                                        <auc:ReheatPlantID IDref="HeatingPlant-A" />
+                                    </auc:CentralAirDistribution>
+                                </auc:DeliveryType>
+                                <auc:HeatingSourceID IDref="HeatingSource-A" />
+                                <auc:CoolingSourceID IDref="CoolingSource-A" />
                                 <auc:Capacity>123</auc:Capacity>
                                 <auc:CapacityUnits>gpm</auc:CapacityUnits>
                                 <auc:YearInstalled>2020</auc:YearInstalled>
+                                <auc:Quantity>1</auc:Quantity>
+                                <auc:DeliveryCondition>Good</auc:DeliveryCondition>
+                            </auc:Delivery>
+                            <auc:Delivery ID="Delivery-B">
+                                <auc:DeliveryType>
+                                    <auc:ZoneEquipment>
+                                        <auc:FanBased>
+                                            <auc:FanBasedDistributionType>
```

Comment:
@nllong for `auc:FanBasedDistributionType`s do we require that the also have an `auc:FanSystem` declared and linked?

Reply:
Yes, that would seem like a great assertion.